Source/ACE.Server/Factories/LootGenerationFactory.cs

```diff
@@ -2250,5 +2250,160 @@ private static float GetTotalProbability(List<TreasureMaterialGroups> list)
             return totalSum;
         }
 
-    }
+        /// <summary>
+        /// Will return correct meleeMod for bow wields (some debate on what 375 top out at, leaving at 18 for now). HarliQ 11/17/19
+        /// </summary>
+        private static double GetWieldReqMeleeDMod(int wield)
+        {
+            double meleeMod = 0;
```

Comment:
this should not have the changes from 2452 merged in

should just be TestLootGen + 2451